cvc5_pythonic_api/cvc5_pythonic.py

```diff
@@ -1727,6 +1734,793 @@ def Or(*args):
+        """
+        return BoolRef(
+            self.ctx.solver.mkTerm(Kind.STRING_LT, self.ast, other.ast), self.ctx
+        )
+
+    def __ge__(self, other):
+        """Create the SMT expression `other <= self` based on a lexiographic ordering.
+
+        >>> s,t = Strings('s t')
+        >>> s >= t
+        t <= s
+        """
+        return BoolRef(
+            self.ctx.solver.mkTerm(Kind.STRING_LEQ, other.ast, self.ast), self.ctx
+        )
+
+    def __gt__(self, other):
+        """Create the SMT expression `other < self` based on a lexiographic ordering.
+
+        >>> s,t = Strings('s t')
+        >>> s > t
+        t < s
+        """
+        return BoolRef(
+            self.ctx.solver.mkTerm(Kind.STRING_LT, other.ast, self.ast), self.ctx
+        )
+
+
+def StringSort(ctx=None):
+    """Return the string sort in the given context. If `ctx=None`, then the global context is used.
+
+    >>> StringSort()
+    String
+    >>> a = Const('a', StringSort())
+    >>> a.sort() == StringSort()
+    True
+    """
+    ctx = _get_ctx(ctx)
+    return StringSortRef(ctx.solver.getStringSort(), ctx)
+
+
+def String(name, ctx=None):
+    """Return a string constant named `name`. If `ctx=None`, then the global context is used.
+
+    >>> s = String('s')
+    >>> s.sort()
+    String
+    """
+    ctx = _get_ctx(ctx)
+    e = ctx.get_var(name, StringSort(ctx))
+    return StringRef(e, ctx)
+
+
+def Strings(names, ctx=None):
+    """Return a tuple of string constants.
+
+    >>> x, y, z = Strings('x y z')
+    >>> Concat(x,y,z)
+    +(x, y, z)
+    >>> Concat(x,y,z).sort()
+    String
+    """
+    ctx = _get_ctx(ctx)
+    if isinstance(names, str):
+        names = names.split(" ")
+    return [String(name, ctx) for name in names]
+
+
+def SeqSort(s):
+    """Create a sequence sort over elements provided in the argument
+    >>> s = SeqSort(IntSort())
+    >>> s == Unit(IntVal(1)).sort()
+    True
+    """
+    return SeqSortRef(s.ctx.solver.mkSequenceSort(s.ast), s.ctx)
+
+
+def Empty(s):
+    """Create the empty sequence of the given sort
+
+    >>> e = Empty(StringSort())
+    >>> e2 = StringVal("")
+    >>> print(e.eq(e2))
+    True
+    >>> e3 = Empty(SeqSort(IntSort()))
+    >>> print(e3)
+    (as seq.empty (Seq (Seq Int)))()
+    """
+    if isinstance(s, ReSortRef):
+        return ReRef(s.ctx.solver.mkRegexpNone(), s.ctx)
+    if isinstance(s, StringSortRef):
+        return StringRef(s.ctx.solver.mkString(""), s.ctx)
+    return _to_expr_ref(s.ctx.solver.mkEmptySequence(s.ast), s.ctx)
+
+
+def is_seq(a):
+    """Return `True` if `a` is a sequence expression.
+
+    >>> print (is_seq(Unit(IntVal(0))))
+    True
+    >>> print (is_seq(StringVal("abc")))
+    True
+    """
+    return isinstance(a, SeqRef)
+
+
+def is_string(a):
+    """Return `True` if `a` is a string expression.
+
+    >>> x = Int('x')
+    >>> a = String('a')
+    >>> is_string(x)
+    False
+    >>> is_string(a)
+    True
+    """
+    return isinstance(a, StringRef)
```

Comment:
You should add a top-level function `is_string_value`: https://github.com/Z3Prover/z3/blob/master/src/api/python/z3/z3.py#L10967